content/es/working-groups.md

```diff
@@ -0,0 +1,279 @@
+
+Es importante que los GT no se formen prematuramente. Los grupos de trabajo no son formados para *comenzar* una serie de tareas si no más bien formados una vez que el trabajo se esta llevando a cabo y los colaboradores piensan que es beneficioso hacerse como un proyecto autónomo.
+
+Si el trabajo definido en la carta del Grupo de Trabajo se completa, el Grupo debe disolverse y la responsabilidad de gobernación absorbida de nuevo por el CT.
+
+## Grupos de Trabajo actuales
+
+* [Website](#website)
+* [Streams](#streams)
+* [Build](#build)
+* [Tracing](#tracing)
+* [i18n](#i18n)
+* [Evangelism](#evangelism)
+* [Roadmap](#roadmap)
+* [Docker](#docker)
+* [Iniciar un Grupo de Trabajo](#starting-a-wg)
+* [Bootstrap Governance](#bootstrap-governance)
+
+### [Website](https://github.com/iojs/website)
+
+El propósito del grupo de trabajo del website es el de construir y mantener un sitio público para el proyecto de `io.js`.
+
+Sus responsabilidades son:
+* Desarrollar y mantener un sistema automatizado para `iojs.org`.
+* Asegurarse de que el sitio se actualiza regularmente con cambios hechos en `io.js`, tales como publicaciones y nuevas funciones.
+* Acoger y habilitar una comunidad de traductores.
+
+### [Streams](https://github.com/iojs/readable-stream)
+
+El GT Streams está dedicado a dar soporte y mejorar la API de Streams como es
+usada en io.js y el sistema npm. Buscamos crear una API capaz de componerse que resuelva el problema presentado por multiples ocurrencias de un evento a lo largo del tiempo de una forma humana y con poca complejidad. Las mejoras en la API vendrán determinadas por las necesidades del ecosistema; interoperabilidad y compatibilidad con otras soluciones y versiones anteriores es de vital importancia. Nuestras responsabilidades incluyen:
+
+* Hacer frente a los stream issues en el io.js issue tracker.
+* Autoría y edición de la documentación de streams perteneciente al proyecto io.js.
+* Revisar cambios a las subclases de streams subclasses pertenecientes al proyecto io.js.
+* Redireccionar cambios de streams desde el proyecto io.js project a éste.
+* Asistir en la implementación de proveedores de streams dentro de io.js.
+* Recomendar que versiones de readable-stream han de ser incluidas en io.js.
+* Proveer mensajes acerca del futuro de streams para dar a la comunidad avisando a la comunidad de los cambios.
+
+### [Build](https://github.com/iojs/build)
+
+El propósito del Grupo de Trabajo build es el de crear y mantener una infraestructura automatizada.
+
+Sus responsabilidades son:
+* Producir Paquetes para todas las plataformas objetivo.
+* Realizar tests.
+* Realizar tests de rendimiento y comparaciones.
+* Crea y gestiona build-containers.
+
+### [Tracing](https://github.com/iojs/tracing-wg)
+
+El propósito del Grupo de Trabajo tracing es el de aumentar la transparencia del software escrito en io.js.
```

Comment:
Same reason as I wrote to @ipeluffo. 

Is defined before but I left it just in case is the first thing read. The reason for this to happen is that titles are links when translated from markdown to html. It would create confusion if you don't correctly guess what GT is.

I've decided to only use the acronym if occurs more than once in a section or it was defined already on a section.